web/js/feedback.js

```diff
@@ -7,8 +7,7 @@
 
 import util from './util/util';
 
-export const feedbackModal = (function () {
-  var self = {};
+export default (function (self) {
```

Comment:
Why go from a named export to an unnamed `default` export?

Reply:
Discussed in slack chat; export constant does not work in this case. We can look at changing this to a named function when we do a review. Util functions are setup in a similar manner as this.